lib/pool.py

```diff
@@ -136,3 +136,11 @@ def install_custom_uefi_certs(self, auths):
             host.ssh(['secureboot-certs', 'install'] + params)
         finally:
             host.ssh(['rm', '-f'] + list(auths_dict.values()))
+
+    def eject_host(self, host):
+        master = self.master
+        master.xe('pool-eject', {'host-uuid': host.uuid, 'force': True})
+        wait_for_not(lambda: host.uuid in self.hosts_uuids(), "Wait for host {host} to be ejected of pool {master}.")
```

Comment:
Not entirely sure but I would write "from".

```suggestion
        wait_for_not(lambda: host.uuid in self.hosts_uuids(), "Wait for host {host} to be ejected from pool {master}.")
```